src/libraries/System.Private.Xml/src/System/Xml/XPath/Internal/ClonableStack.cs

```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+#nullable enable
```

Comment:
should you mark T as `[MaybeNull]`?

Reply:
Generic types are already inferred as `MaybeNull`. Is that what you were referring to?
![image](https://user-images.githubusercontent.com/16040868/88333496-319a6300-cce5-11ea-845c-b58de1e6c608.png)